src/commands/setup.yml

```diff
@@ -0,0 +1,31 @@
+description: >
+      Create ECR repository for docker image out of the project and create SQS queues for deployments
+      $AWS_ACCOUNT_ID must be set
+
+parameters:
+      env:
+            description: target environment (i.e, qa, staging, production)
+            type: string
+      project:
+            description: your git repository project name
+            type: string
+      bucket:
+            description: S3 bucket name
+            type: string
+
+steps:
+      - run:
+            name: Install aws cli
+            command: sudo apt install awscli -y
+
+      - run:
+            name: "Create SQS queues"
+            command: |
+              aws s3 cp s3://<< parameters.bucket >>/sqs-attributes.json .
```

Comment:
also, you should check if the queue exist before creating, no ? don't we plan to run this every time ?

Reply:
It's ok to create every time, tested. if queue not exist it will be created and if exist this will return queue details, will not fail and will not create the new one and will not update.